llama_index/agent/utils.py

```diff
@@ -15,3 +19,23 @@ def add_user_step_to_memory(
     memory.put(user_message)
     if verbose:
         print(f"Added user message to memory: {step.input}")
+
+
+def create_agent_from_llm(
```

Comment:
@logan-markewich I would prefer something like `AgentRunner.from_llm`, but I didn't want to add circular references - any suggestions?

Reply:
I think this is fine tbh

Reply:
@logan-markewich writing the doc, using `create_agent_from_llm` didn't feel right, so I now used `AgentRunner.from_llm` with dynamic imports (to avoid cyclic dependencies). Seems to work nicely - IMHO good to go.